doc/site/modularity.markdown

```diff
@@ -318,7 +318,7 @@ for (i in 1..2) {
 System.print(3)    //>  not reached
 </pre>
 
-Although it is not invalid to return a value, there is no way to access that value and it is therefore simply discarded.
+Although it is not invalid to return a value, it can only be accessed using meta-programming - reading the file contents and compiling them with the Meta.compile method. Otherwise it would simply be discarded.
```

Comment:
https://github.com/wren-lang/wren/issues/974#issuecomment-821867000

Reply:
It's true in the sense that you **can** access the return value this way. Whether anyone would go to these lengths to do so, I rather doubt. We certainly can't leave the wording as it stands because it's just not true to say there's no way to access it.

Reply:
Well, there is a simpler way:
```wren
System.print(Fn.new {
  return 1
}.call())
```
Yay! I can access the return value of... Functions?

This is exactly what `Meta.compile()` does: return a function with the specified code.

Quoting https://wren.io/modules/meta/meta.html#compile(source):

> Compiles Wren source code into a closure and then returns the closure

Reply:
How about saying instead:

Although it is not invalid to return a value, there is no **direct** way to access that value and it is therefore simply discarded.

Reply:
No direct neither indirect.

I will be happy to see a statement about using the return value in the documentation of `Meta.compile()`, but here it's just wrong, both technically and practically.